private-templates-service/client/src/store/page/actions.ts

```diff
@@ -0,0 +1,9 @@
+import { PageAction, NAVIGATION } from "./types";
+
+export function setPage(currentPage: string): PageAction{
+  return{
+    type: NAVIGATION,
+    text: "The user has changed the page",
```

Comment:
Unless I'm missing something I dont think you need descriptions in your actions

Reply:
We're leaving this in to make it easier to debug with the redux plugin on chrome